src/Tools/Microsoft.dotnet-openapi/src/Commands/BaseCommand.cs

```diff
@@ -272,6 +275,56 @@ internal async Task<string> DownloadGivenOption(string url, CommandOption fileOp
             }
         }
 
+        /// <summary>
+        /// Retries every 1 sec for 60 times by default.
+        /// </summary>
+        /// <param name="retryBlock"></param>
+        /// <param name="logger"></param>
+        /// <param name="cancellationToken"></param>
+        /// <param name="retryCount"></param>
+        private static async Task<IHttpResponseMessageWrapper> RetryRequest(
+            Func<Task<IHttpResponseMessageWrapper>> retryBlock,
+            CancellationToken cancellationToken = default,
+            int retryCount = 60)
```

Comment:
This is a straight copy of https://github.com/aspnet/AspNetCore/blob/425c196cba530b161b120a57af8f1dd513b96f67/src/Hosting/Server.IntegrationTesting/src/Common/RetryHelper.cs because that isn't a shipping package. I don't think it's valuable to change this copy since that one works just fine.